libvast/src/system/configuration.cpp

```diff
@@ -415,28 +435,7 @@ caf::error configuration::parse(int argc, char** argv) {
   auto [ec, it] = plugin_opts.parse(content, plugin_args);
   VAST_ASSERT(ec == caf::pec::success);
   VAST_ASSERT(it == plugin_args.end());
-  // Gather and parse all to-be-considered configuration files.
-  std::vector<std::string> cli_configs;
-  for (auto& arg : command_line)
-    if (arg.starts_with("--config="))
-      cli_configs.push_back(arg.substr(9));
-  if (auto configs = collect_config_files(config_dirs(*this), cli_configs))
-    config_files = std::move(*configs);
-  else
-    return configs.error();
-  auto config = load_config_files(config_files);
-  if (!config)
-    return config.error();
-  *config = flatten(*config);
-  if (auto err = merge_environment(*config))
-    return err;
-  // From here on, we go into CAF land with the goal to put the configuration
-  // into the members of this actor_system_config instance.
-  auto settings = to_settings(std::move(*config));
-  if (!settings)
-    return settings.error();
-  if (auto err = embed_config(*settings))
-    return err;
+  fmt::print(stderr, "{}\n", caf::deep_to_string(content));
```

Comment:
:eyes:

Reply:
Yeah, one of the things that's just easier with good ol' printf debugging. Will remove.

Reply:
I force-pushed. You saw nothing!